sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ValueStateImplWithTTL.scala

```diff
@@ -165,7 +151,10 @@ class ValueStateImplWithTTL[S](
    * grouping key.
    */
   private[sql] def getValuesInTTLState(): Iterator[Long] = {
```

Comment:
Now this should only be at most one value - let's return `Option[Long]` and update the method doc.